packages/relayer-lib/src/tx_builder/eth_to_cosmos.rs

```diff
@@ -141,7 +147,7 @@ impl<P: Provider + Clone> TxBuilder<P> {
         ethereum_consensus_state: &ConsensusState,
     ) -> Result<()> {
         wait_for_condition(
-            Duration::from_secs(60 * 10),
+            Duration::from_secs(60 * 30),
```

Comment:
why was it increased to 30 mins

Reply:
Same as I mentioned before, to enable mainnet/sepolia runs